brunch-config.js

```diff
@@ -31,8 +31,7 @@ exports.config = {
   plugins: {
     babel: {
       // Do not use ES6 compiler in vendor code
-      ignore: [/^(web\/static\/vendor)/],
-      loose: "all"
```

Comment:
@chrismccord Found the [commit](https://github.com/phoenixframework/phoenix/commit/41bfcf314ae3dd6a9fb7e1231ac0db7ba1def5e1) by @ericmj on 17 February 2015. The commit message includes the following:

> Generate phoenix.js with the “Loose Mode” option on es6to5 which in conjunction with es5-shim makes the javascript work on IE8.

I can jumpstart a VM, this weekend, to test it on IE8, if that browser is still a concern for the development team.